File: dart/dart-find-sum-list-elements/example2.dart
// https://codevscolor.com/dart-find-sum-list-elements
main() {
  var sum = 0;
  var given_list = [1, 2, 3, 4, 5];
  var i = 0;

  while (i < given_list.length) {
    sum += given_list[i];
    i++;
  }

  print("Sum: ${sum}");
}

File: javascript/javascript-find-sum-even-below-number/example2.js
// https://codevscolor.com/javascript-find-sum-even-below-number

function isEven(n) {
  return n % 2 == 0;
}

function findSum(no) {
  let sum = 0;
  let i = 1;

  while (i <= no) {
    if (isEven(i)) {
      sum += i;
    }
    i++;
  }
  return sum;
}

console.log(findSum(100));
